pkg/util/callback.go

```diff
@@ -39,6 +39,10 @@ func (c *callbackInvoker) ExecuteCallback(callbackURL string, success bool, mess
 		"success": success,
 	}
 
+	for name, value := range labels {
+		d[name] = value
+	}
```

Comment:
My concern was that at some point the labels would conflict with the other entries we add to the map